ClassLibrary/Employee.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace ClassLibrary
+{
+    public class Employee
+    {
+        private string firstName;
+        private string lastName;
+        private int age;
+        private int experienceYears;
+        private string position = "Intern";
+
+        public string getPosition
+        {
+            get { return position; }
+            set { position = value; }
+        }
+        public Employee()
+        {
+            firstName = "New";
+            lastName = "Employee";
+        }
+        public Employee(string firstName, string lastName, int age)
+        {
+            this.firstName = firstName;
+            this.lastName = lastName;
+            this.age = age;
+        }
+        public Employee(string firstName, string lastName, int age, int experienceYears, string position)
+        {
+            this.firstName = firstName;
+            this.lastName = lastName;
+            this.age = age;
+            this.experienceYears = experienceYears;
+            this.position = position;
+        }
+
+        public override string ToString()
+        {
+            return $"{firstName} {lastName} | Age: {age} | Experience Years: {experienceYears} | Position: {position}";
+        }
+
+        public void VacationDays()
```

Comment:
this method is not used anywhere. Use it somewhere in code
Also let's add one more method which will calculate a work coefficient based on age and experience year

Reply:
done (I believe so:)